orthopairs/src/main/java/org/reactome/release/orthopairs/OrthologyFileParser.java

```diff
@@ -0,0 +1,119 @@
+package org.reactome.release.orthopairs;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.json.simple.JSONObject;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+
+public class OrthologyFileParser {
```

Comment:
It might be better to make this class use non-static fields and methods since the mapping data processed is being stored.  It makes more sense for that to belong to an instance of the class rather than the class itself.

You could add a constructor something like this:

```
public OrthologyFileParser(List<String> pantherFiles, String sourceMappingSpecies, JSONObject speciesJSONFile) {
    try {
        parsePantherOrthologFiles(pantherFiles, sourceMappingSpecies, speciesJSONFile);
    } catch (IOException e) {
        logger.fatal("Could not parse Panther Ortholog files", e);
	throw new RuntimeException(e);
    }
}
```